nmrfx-processor/src/main/java/org/nmrfx/processor/math/Vec.java

```diff
@@ -2049,9 +2040,7 @@ public Vec bcWhit(double lambda, int order, boolean baselineMode) {
             //is this fine?
             Util.asmooth(w, y, z, a, lambda, vecSize, order);
             if (baselineMode) {
-                for (int i = 0; i < vecSize; i++) {
-                    rvec[i] = z[i + 1];
-                }
+                if (vecSize >= 0) System.arraycopy(z, 1, rvec, 0, vecSize);
```

Comment:
It is my preference to always have {} even if statements are a single line to reduce changes of bugs in the future, but I don't know if that is a standard java convention or not, or if @vfiack has a preference about it

Reply:
I prefer {} as well.  All these single lines were introduced by IntelliJ when it suggested using System.arraycopy.  I think I"ve fixed them all now.

Reply:
I think that might be something you can adjust in settings so it doesn't do that, but there seems to be two possible places related to that, so I am not sure which one would need to be set.